### PR TITLE
Added layout support

### PR DESCRIPTION
**Pull request by matzayonc**

### CODE REVIEW

File: crates/common/src/inputs/cairo_0_prover_input.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ use crate::ProverInput;
 pub struct Cairo0ProverInput {
     pub program: CompiledProgram,
     pub program_input: serde_json::Value,
+    pub layout: serde_json::Value,
```
Comment (neotheprogramist) [inline crates/common/src/inputs/cairo_0_prover_input.rs:9]: can be just simple `String`?

File: crates/common/src/inputs/cairo_1_prover_input.rs [hunk 2]
```diff
@@ -6,6 +6,7 @@ use crate::ProverInput;
 pub struct Cairo1ProverInput {
     pub program: Cairo1CompiledProgram,
     pub program_input: serde_json::Value,
+    pub layout: serde_json::Value,
```
Comment (neotheprogramist) [inline crates/common/src/inputs/cairo_1_prover_input.rs:9]: can be just simple String?

**Review by neotheprogramist — CHANGES_REQUESTED** [review]

File: crates/common/src/inputs/cairo_0_prover_input.rs (hunk 1 above)
Comment (matzayonc) [reply crates/common/src/inputs/cairo_0_prover_input.rs]: it can

File: crates/common/src/inputs/cairo_1_prover_input.rs (hunk 2 above)
Comment (matzayonc) [reply crates/common/src/inputs/cairo_1_prover_input.rs]: it is now

**Review by neotheprogramist — APPROVED** [review]
LGTM

**Pull request merged**